README.md

```diff
@@ -48,11 +48,9 @@ to the full API surface, including deprecated endpoints.
 The SDK API is 100% compatible with the Kubernetes API, and is
 schematically identical to what Kubernetes users expect.
 
-At a minimum, we support the [current Kubernetes version](https://kubernetes.io/docs/setup/release/) and the previous two versions.
-Older versions will likely work as well, but are not officially supported.
-
-See the [CHANGELOG](https://github.com/pulumi/pulumi-kubernetes/blob/master/CHANGELOG.md) for details
-on supported versions of Kubernetes for each version of the `pulumi-kubernetes` package.
+At a minimum, we support the [current Kubernetes version](https://kubernetes.io/docs/setup/release/)
+and the previous two versions. Although not tested regularly against older versions, this provider
+should work with any Kubernetes cluster with version >1.9.0.
```

Comment:
What *do* we test?  Is there any reason not to just test against 1.9.0 and then say "we support all versions >1.9.0"?

Reply:
We currently test against the latest GKE version. The difficult part for testing is standing up an ephemeral k8s cluster for any versions not available on GKE.

Reply:
Just checked, and this currently includes 1.12-1.14. While it would be possible for us to create older cluster versions with other tools rather than using a managed service, that would be a significant amount of engineering effort. I don't think it's worth the cost at this time.

Reply:
Given that we don't test on 1.17, 1.16 or 1.15 either though - I'm not sure I'd make a point of "not tested regularly against older versions" (since arguably we only test old versions).

I think it's honestly going to be simplest to just way we support version 1.9.0 and higher. 

I think we should open an issue to track testing against a broader range of Kubernetes versions (including both older and probably more importantly newer) versions.  The one way I'd want to strengthen the above statement is to call our what range of versions we regularly test on - but sounds like we don't have a broad enough matrix to make any particularly helpful statement about that currently?